README.md

```diff
@@ -3,16 +3,22 @@
 This repo holds information about subnet chains that exist on the Avalanche
 network.
 
+## To add your subnet to this list, please make a pull request that adds the following: 
+
 Each chain in the `chains` folder may contain the following files:
-- `chain-information.json`: General information about the chain such as the
-  name, vm configuration, and token configuration.
+- `genesis.json`: To get started, use the
```

Comment:
Can we modify this to make it clear avalanche-cli is one option as opposed to a requirement?

Reply:
Yes, I just assumed that if someone who makes custom changes to the genesis knows what they are doing while someone looking create a draft genesis will use Avalanche-CLI. This is why I don't include a Genesis template, so that teams don't include features they don't need.